README.md

```diff
@@ -1,53 +1,57 @@
+[![Website cusdeb.com](https://img.shields.io/website-up-down-green-red/https/lbesson.bitbucket.io.svg)](https://cusdeb.com)
+
 # CusDeb Web Client
 
-CusDeb Web Client is a web client which relies on such CusDeb services as [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api), [Black Magic](https://github.com/tolstoyevsky/blackmagic), [Dominion](https://github.com/tolstoyevsky/dominion) and [Orion](https://github.com/tolstoyevsky/orion).
+CusDeb Web Client exposes the entirety of CusDeb's functionality to the user, including customizing OS images for wide variety of single-board computers.
 
 ## Table of Contents
 
-- [Requirements](#requirements)
 - [Configuration](#configuration)
-- [Building](#building)
+- [Development](#development)
 - [Authors](#authors)
 - [Licensing](#licensing)
 
-## Requirements
-
-CusDeb Web Client requires
-
-* running [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api) and [Black Magic](https://github.com/tolstoyevsky/blackmagic);
-* [Node.js](https://nodejs.org/) 12 or above.
-
 ## Configuration
 
 CusDeb Web Client can be configured via the following environment variables (called parameters).
 
-| Parameter        | Description |
-|------------------|-------------|
-| `BM_RPC_URL`     | The URL of the WebSocket API provided by Black Magic. If Black Magic listens on the port 8002, the value of the param *might* look like `ws://127.0.0.1:8002/rpc/token/%token` when running locally. |
-| `CUSDEB_API_URL` | The URL of CusDeb API, including its version. The value of the param *must be* `/api/v1` when running locally. |
-| `HOST`           | The address of the machine Webpack Dev Server is running on. It helps to check CusDeb Web Client on other devices in a local network. |
+| Parameter              | Description                                                                                                                                         | Default                                     |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|
+| `BLACKMAGIC_URL`       | The URL of the WebSocket API provided by [Black Magic](https://github.com/tolstoyevsky/blackmagic).                                                 | `ws://localhost:8002/bm/token/%token`       |
+| `CUSDEB_ANONYMOUS_URL` | The URL of [CusDeb Anonymous](https://github.com/tolstoyevsky/cusdeb-anonymous).                                                                    | `http://localhost:8007`                     |
+| `CUSDEB_API_URL`       | The URL of [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api).                                                                                | `http://localhost:8001`                     |
```

Comment:
Let's paraphrase the description: `The address of the [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api) service`